site/enterprise-faq.html

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
I'd like to get David to design this page out. This layout feels thrown together and it doesn't match our website visual design and branding. I think it should have a main hero like the homepage and the background image should go along with some type of sales theme.

I'd also like to pull the FAQ up and put it right below the header. 

There customer section could stay or maybe we toss it because it doesn't have much to-do with the page.

Finally, if David feels like the hero image is not something he wants to tackle, we can have Guiceworks do it. They did the website design originally.